packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -520,6 +567,112 @@ export class AmqpConnection {
     return consumerTag;
   }
 
+  private async setupBatchSubscriberChannel<T>(
+    handler: BatchSubscriberHandler<T>,
+    msgOptions: MessageHandlerOptions,
+    channel: ConfirmChannel,
+    originalHandlerName = 'unknown',
+    consumeOptions?: ConsumeOptions
+  ): Promise<ConsumerTag> {
+    const queue = await this.setupQueue(msgOptions, channel);
+    const batchSize = msgOptions.batchOptions?.size ?? 10;
```

Comment:
Please update the JSDOC to include the defaults `10` for batchSize and `200` for timeout as we don't want to hide them from public API.